server/integration/jgroups/src/main/java/org/infinispan/server/jgroups/ManagedSocketFactory.java

```diff
@@ -80,7 +80,7 @@ private Socket register(final Socket socket) {
 
     @Override
     public ServerSocket createServerSocket(String name) throws IOException {
-        return this.register(this.factory.createServerSocket(name));
+        return this.factory.createServerSocket(name);
```

Comment:
I recall making changes to wildfly-core to address this.  Let me find the relevant upstream commit.